ocs_ci/helpers/helpers.py

```diff
@@ -2790,3 +2790,33 @@ def fetch_used_size(cbp_name, exp_val=None):
             f"matching. Retrying"
         )
     return used_in_gb
+
+
+def verify_cli_cmd_output(
```

Comment:
This function not only verifies the output but also executes the command. Is there a way we can maybe incorporate the functionality of output validation within the existing `run_cmd` function? Maybe by adding an arg to `run_cmd` for output validation true or false.

Reply:
I added `logger.info(out)`

Reply:
what I meant is that maybe you can use the already existing `run_cmd()` function and enhance it to do the validation you intend to do with this new function